editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx

```diff
@@ -186,14 +255,135 @@ function navigatorStructure(editorState: EditorState, deriveState: DerivedState)
   return lines.join('\n')
 }
 
-xdescribe('conditionals in the navigator', () => {
+describe('conditionals in the navigator', () => {
   setFeatureForBrowserTests('Conditional support', true)
   setFeatureForBrowserTests('Fragment support', true)
-  it('dragging into a non-empty clause, creates a fragment wrapper', async () => {
+  xit('dragging into a non-empty clause, creates a fragment wrapper', async () => {
     // TODO: Fill this out.
   })
-  it('dragging into an empty clause, creates a fragment wrapper', async () => {
+  it('dragging into an empty active clause, takes the place of the empty value', async () => {
     FOR_TESTS_setNextGeneratedUids([
+      'conditional2',
```

Comment:
What's the deal with this huge list of alternating UIDs? This strikes me as something that's going to be a nightmare to maintain, because it's not clear which interactions below will be triggering new UID generation (and how many new UIDs are needed each time).

Reply:
Without spending a bunch of time getting them down to exactly the number of entries needed I mostly did this for expedience, especially as I was using the tests from the start. It's just a bit awkward because it's not super intuitive how many you do need without doing it by trial and error.

Reply:
Improved these in 154326c26.